indico/modules/rb/controllers/backend/bookings.py

```diff
@@ -345,11 +358,27 @@ def _process(self, args):
             additional_booking_attrs['new_booking_id'] = new_booking.id
 
         db.session.flush()
+        self.after_update(args, reservation=self.booking, new_split_reservation=new_booking)
```

Comment:
and for htis we already have a `booking_modified` signal

Reply:
Similarly to the `booking_created` signal above, "extra fields" data need to be passed to the signal somehow. In this case it is a bit more complicated than with create, because the `Reservation.modify` [method](https://github.com/indico/indico/blob/c87a07e1d68082364c1427e4b377ca485cd05547/indico/modules/rb/models/reservations.py#L604C13-L604C13) where the signal is dispatched does not have access to the complete form data. I suppose adding `args` to the function arguments [here](https://github.com/indico/indico/blob/c87a07e1d68082364c1427e4b377ca485cd05547/indico/modules/rb/controllers/backend/bookings.py#L339C30-L339C30) and [here](https://github.com/indico/indico/blob/c87a07e1d68082364c1427e4b377ca485cd05547/indico/modules/rb/controllers/backend/bookings.py#L344) and then passing them to the `Reservation.modify` and from there to the signal would work?

Reply:
Hello @ThiefMaster, thanks for the [answer](https://github.com/indico/indico/pull/6126#discussion_r1455150168) above. Do you think the solution outlined here is also better than the current solution?

Reply:
Yes, I think so.